rules/common_settings.bzl

```diff
@@ -75,24 +75,26 @@ def _string_impl(ctx):
     if len(allowed_values) == 0 or value in ctx.attr.values:
         return BuildSettingInfo(value = value)
     else:
-      fail("Error setting "+ str(ctx.label) + ": invalid value '" + value + "'. Allowed values are " + str(allowed_values))
+        fail("Error setting " + str(ctx.label) + ": invalid value '" + value + "'. Allowed values are " + str(allowed_values))
```

Comment:
is this longer than 100 chars? it's going to complain internally

Reply:
buildifier did it. The same bits were checked in as bazelbuild/bazel/tools/config/common_settings.bzl and it did not complain then.

Reply:
Actually, it should not complain. Vendored in code is not subject to line length restrictions. It would be impossible to import stuff if that were true.

Reply:
There's no line length limit.